src/main/java/net/sf/jabref/logic/mods/MODSEntry.java

```diff
@@ -116,11 +116,11 @@ private void populateFromBibtex(BibEntry bibtex) {
         if (bibtex.hasField(BibEntry.KEY_FIELD)) {
             id = bibtex.getCiteKey();
```

Comment:
Use the Optional methods getFiedOptional 
Then you can use the ifPresent or isPresent to check if an Entry is there